### PR TITLE
Update domains.json

### PR DESCRIPTION
**Pull request by athamefinance**
athame finance

#### Hi! What is your PR about?

- [ ] Add or edit a skin
- [X] Add a custom domain for your space
- [ ] Add an alias to migrate your space

### CODE REVIEW

**Review by gokaiorg — CHANGES_REQUESTED** [review]
There is a conflict in your PR since `domains.json` file has been updated, can you resolve it?
To prevent conflicts, it is recommended to add your subdomain between two existing domains rather than at the end or beginning of the list
].

**Comment by athamefinance** [comment]
I did actually add in between :}

File: spaces/domains.json [hunk 1]
```diff
@@ -617,7 +617,8 @@
   "vote.eggsforukraine.com": "eggsforukraine.eth",
   "vote.komorebinosekai.com": "komorebi-no-sekai.eth",
   "dao.getagump.org": "getagump.eth",
+  "vote.athame.finance":  "athamefinance.eth",
```
Comment (gokaiorg) [inline spaces/domains.json:620]: Please could you remove the extra space between key and value?
```suggestion
  "vote.athame.finance": "athamefinance.eth",
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Comment by gokaiorg** [comment]
To update your fork with the remote version of snapshot-spaces you can try these GIT commands then commit your changes again.
`git remote add upstream https://github.com/snapshot-labs/snapshot-spaces.git`
`git fetch upstream`
`git checkout master`
`git merge upstream/master`

**Pull request closed without merging**